### PR TITLE
Update changelog for release 0.3.0

### PR DESCRIPTION
**Pull request by jleibs**
After a successful launch a couple of weeks ago, we're back with our first release!
With a few exceptions this release focuses on internal refactors & improving our processes.
However, we think you'll enjoy these goodies that made it in nonetheless!

https://user-images.githubusercontent.com/2910679/222510504-23871b8c-0bef-49c2-bbd2-37baab4247e8.mp4

* You can now generate point clouds directly from depth textures! Check out @teh-cmc's [video](https://www.youtube.com/watch?v=7eJWS_DGBn0) on how to use it.
  * This is **a lot** faster and more convenient than doing so manually in your own code
  * Allows to choose you from a wide variety of color maps
  * Picking is not yet working
* Python packages now work with Ubuntu-20.04 [#1334](https://github.com/rerun-io/rerun/pull/1334)
* Improve display of date-times in plots [#1356](https://github.com/rerun-io/rerun/pull/1356)
* UI scale factor is saved across sessions and more persistent between screens [#1448](https://github.com/rerun-io/rerun/pull/1448)
* SDK:
  * 2D Line strips can now be logged directly [#1430](https://github.com/rerun-io/rerun/pull/1430)
  * Add a `strict` mode to the Python SDK where misuses of the API results in exceptions being raised.[#1477](https://github.com/rerun-io/rerun/pull/1477)
* Showing tensors in the viewer is now faster
* Log streams / .rrd files are now versioned [#1420](https://github.com/rerun-io/rerun/pull/1420)
    * ⚠️ BREAKING: old `.rrd` files no longer load
* Less build dependencies ([still more work to do!](https://github.com/rerun-io/rerun/issues/1316))
  * notably, we previously used a version of the `time` crate which had a security issue (CVE-2020-26235)
* Print more information & troubleshooting info on crash
* u8 segmentation stay u8 now (they converted to u16 before) [#1376](https://github.com/rerun-io/rerun/pull/1376)

---

* Windows
  * Web "runs" but still subject to https://github.com/rerun-io/rerun/issues/1195
  * [x] Python Wheel
    * [x] Web
    * [x] Native
  * [x] Rust crate
    * [x] Web
    * [x] Native
  * [x] Rust install
    * [x] Web
    * [x] Native
* Linux
  * [ ] Python Wheel
    * [ ] Web
    * [ ] Native
  * [ ] Rust crate
    * [ ] Web
    * [ ] Native
  * [ ] Rust install
    * [ ] Web
    * [ ] Native
* Mac
  * [x] Python Wheel
    * [x] Web
    * [x] Native
  * [x] Rust crate
    * [x] Web
    * [x] Native
  * [x] Rust install
    * [x] Web
    * [x] Native

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -16,37 +16,37 @@ include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
...
-re_smart_channel = { path = "crates/re_smart_channel", version = "0.2.0" }
-re_string_interner = { path = "crates/re_string_interner", version = "0.2.0" }
-re_tensor_ops = { path = "crates/re_tensor_ops", version = "0.2.0" }
-re_tuid = { path = "crates/re_tuid", version = "0.2.0" }
-re_ui = { path = "crates/re_ui", version = "0.2.0" }
-re_viewer = { path = "crates/re_viewer", version = "0.2.0" }
-re_web_viewer_server = { path = "crates/re_web_viewer_server", version = "0.2.0" }
-re_ws_comms = { path = "crates/re_ws_comms", version = "0.2.0" }
-rerun = { path = "crates/rerun", version = "0.2.0" }
+re_analytics = { path = "crates/re_analytics", version = "0.3.0-alpha.0" }
```
Comment (emilk) [inline Cargo.toml:26]:
```suggestion
re_analytics = { path = "crates/re_analytics", version = "x=0.3.0-alpha.0" }
```

here and below. See the comment right above this line for why :]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,23 @@
...
+
+Highlights:
+
 * ⚠️ BREAKING: old `.rrd` files no longer load
 
+* Python packages now work with Ubuntu-20.04 [#1334](https://github.com/rerun-io/rerun/pull/1334)
 * Improve display of date-times in plots [#1356](https://github.com/rerun-io/rerun/pull/1356)
-* (this will be filled in once we do a new release)
+* Don't convert u8 segmentation images to u16 [#1376](https://github.com/rerun-io/rerun/pull/1376)
+* Log streams / .rrd files are new versioned [#1420](https://github.com/rerun-io/rerun/pull/1420)
```
Comment (nikolausWest) [inline CHANGELOG.md:16]:
```suggestion
* Log streams / .rrd files are now versioned [#1420](https://github.com/rerun-io/rerun/pull/1420)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,11 +4,152 @@
 ## Unreleased
 [Commits since latest release](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
-* ⚠️ BREAKING: old `.rrd` files no longer load
+## 0.3.0
+### Overview & Highlights
 
+After a successful launch a couple of weeks ago, we're back with our first release!
```
Comment (jleibs) [inline CHANGELOG.md:10]:
```suggestion
After a successful launch a couple of weeks ago, we're back with our second release!
```
I think the launch was the "first" release, right?

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,11 +4,152 @@
...
 * Improve display of date-times in plots [#1356](https://github.com/rerun-io/rerun/pull/1356)
-* (this will be filled in once we do a new release)
+* UI scale factor is saved across sessions and more persistent between screens [#1448](https://github.com/rerun-io/rerun/pull/1448)
+* SDK:
+  * 2D Line strips can now be logged directly [#1430](https://github.com/rerun-io/rerun/pull/1430)
+  * Add a `strict` mode to the Python SDK where misuses of the API results in exceptions being raised.[#1477](https://github.com/rerun-io/rerun/pull/1477)
+* Showing tensors in the viewer is now faster
+* Log streams / .rrd files are now versioned [#1420](https://github.com/rerun-io/rerun/pull/1420)
+    * ⚠️ BREAKING: old `.rrd` files no longer load
+* Less build dependencies ([still more work to do!](https://github.com/rerun-io/rerun/issues/1316))
```
Comment (jleibs) [inline CHANGELOG.md:29]:
```suggestion
* Fewer build dependencies ([still more work to do!](https://github.com/rerun-io/rerun/issues/1316))
```
Nit: fewer for countable things

**Pull request merged**